### DISCUSSION TITLE
Starred tab

### DISCUSSION THREAD

**Discussion by TripleA150** [community/community#138944] [Mobile]
Hi. It would be convenient if the time of the last repository change and the same filters as on the site were added to the starred tab

**Comment by HenSquared** [comment]
Hi @TripleA150 , can you specify the behavior you are hoping for? Maybe screenshots or a video will be helpful. 

Thanks

**Reply by TripleA150** [reply]
Hi. Do it like this, but in the GitHub mobile app. With filters, last repo update and etc.
![image](https://github.com/user-attachments/assets/5ae37d7b-44ed-4535-98e1-7d9ff4103050)
